### ISSUE TITLE
Parsing of HandyLK format

### ISSUE THREAD

**Issue by newca12** [gapt/gapt#22]
_From [bruno...@gmail.com](https://code.google.com/u/105016684496602932564/) on October 07, 2009 22:20:43_

We need to parse from HandyLK format...

_Original issue: http://code.google.com/p/gapt/issues/detail?id=22_

**Comment by newca12** [comment]
_From [shaoli...@gmail.com](https://code.google.com/u/113190107447576027220/) on November 16, 2009 08:40:00_

Bruno, you said once that you would like to replace hlk in some more code oriented
format. I wanted to ask you to specify your ideas here so we can implement it in the
future.

**Comment by newca12** [comment]
_From [shaoli...@gmail.com](https://code.google.com/u/113190107447576027220/) on October 07, 2010 02:26:23_

We will first start with designing the (maybe new) language.

**Comment by newca12** [comment]
_From [shaoli...@gmail.com](https://code.google.com/u/113190107447576027220/) on October 07, 2010 02:29:29_

**Status:** Started  
**Owner:** cdunchev  
**Blockedon:** 80

**Comment by newca12** [comment]
_From [bruno...@gmail.com](https://code.google.com/u/105016684496602932564/) on August 18, 2011 09:07:49_

This issue, together with extraction of projections, is now a bottleneck for GAPT to fully supersede the old CERES system...

**Labels:** -Priority-Medium Priority-Critical

**Comment by newca12** [comment]
_From [fra...@gmail.com](https://code.google.com/u/108596877348066494139/) on August 19, 2011 00:56:08_

Perhaps it would be good to have an issue "Superseding the old CERES system" which will be blocked by all issues that... block it :-)

Another issue that would block it would be Issue 102 .

**Comment by newca12** [comment]
_From [bruno...@gmail.com](https://code.google.com/u/105016684496602932564/) on August 19, 2011 06:23:51_

**Labels:** Milestone-Release2.0

**Comment by newca12** [comment]
_From [bruno...@gmail.com](https://code.google.com/u/105016684496602932564/) on August 19, 2011 12:07:33_

**Labels:** Type-Feature

**Comment by newca12** [comment]
_From [bruno...@gmail.com](https://code.google.com/u/105016684496602932564/) on August 19, 2011 23:15:31_

**Labels:** Component-Parsing

**Comment by newca12** [comment]
_From [cdunc...@gmail.com](https://code.google.com/u/102976883554608245195/) on August 22, 2011 04:52:43_

**Status:** Accepted

**Comment by newca12** [comment]
_From [cdunc...@gmail.com](https://code.google.com/u/102976883554608245195/) on August 22, 2011 04:59:38_

Issue 83 has been merged into this issue.

**Comment by newca12** [comment]
_From [bruno...@gmail.com](https://code.google.com/u/105016684496602932564/) on August 25, 2011 05:51:43_

**Owner:** Martin.R...@googlemail.com

**Comment by newca12** [comment]
_From [Martin.R...@googlemail.com](https://code.google.com/u/113922991942493634983/) on November 30, 2012 06:13:27_

**Labels:** -Priority-Critical Priority-Low  
**Blockedon:** -gapt:80 gapt:80

**Comment by shetzl** [comment]
adding CCs from google code: @quicquid

**Comment by gebner** [comment]
This will be replaced by the tactics language from #241.
